app/models/solidus_paypal_commerce_platform/spree/address_decorator.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module SolidusPaypalCommercePlatform
+  module Spree
+    module AddressDecorator
+      # PayPal doesn't use the phone number, so in cases where the user checks out via
+      # PayPal, this field will be unpopulated. If you want to require phone numbers,
+      # you'll need to turn off cart & product page displays on the payment method edit
+      # page.
+      def require_phone?
+        false && super
```

Comment:
🤦 Yeah you're right, it would never be called. The intent was to allow any custom logic in `require_phone?` to run but ultimately return false... Can't think of any reason why that would be necessary, but better safe than sorry I guess? 😛 `super && false` makes more sense here. Thanks for pointing this out.